added a new section for copyDependencies (#211)
OpenLiberty/guide-jpa-intro: README.adoc

include::finish/backendServices/src/main/java/io/openliberty/guides/event/resources/EventResource.java[]
----

// =================================================================================================
// Configuring the Derby driver and the Liberty
// =================================================================================================

> == Configuring the Derby driver and the Liberty

Reviewer: ```suggestion
== Configuring the Derby driver and the Liberty Maven plugin
```

To use a Derby database, you need to download its libraries and store them to the Liberty shared resources directory. Configure the Liberty Maven plug-in in the `pom.xml` file of the `backendServices` service.

[role="code_command hotspot", subs="quotes"]
----
#Replace the `backendServices/pom.xml` configuration file.#
`backendServices/pom.xml`
----

backendServices/pom.xml
[source, xml, linenums, role='code_column']
----
include::finish/backendServices/pom.xml[]
----

server.xml
[source, Xml, linenums, role='code_column']
----
include::finish/backendServices/src/main/liberty/config/server.xml[]
----

> Three required Derby dependencies are added to the [hotspot=dependencies file=0]`dependencies` configuration for Maven to download the Derby libraries locally. The [hotspot=copyDependencies file=0]`copyDependencies` configuration instructs the Liberty Maven plug-in to copy the Derby libraries to Liberty shared resources directory that is specified through the [hotspot=location file=0]`location` configuration, and is referenced in the [hotspot=shared-dir file=1]`derbyJDBCLib` `library` configration of the Liberty `server.xml` configuration file.

Reviewer: ```suggestion
This configuration adds three required Derby dependencies to the [hotspot=dependencies file=0]`dependencies` configuration so Maven can download the Derby libraries locally. The [hotspot=copyDependencies file=0]`copyDependencies` configuration instructs the Liberty Maven plug-in to copy the Derby libraries to the Liberty shared resources directory that is specified through the [hotspot=location file=0]`location` configuration, and is referenced in the [hotspot=shared-dir file=1]`derbyJDBCLib` `library` configuration of the `server.xml` file.
```
